src/data.py

```diff
@@ -395,6 +398,46 @@ def _assert_summarised_edition_in_dataset(datasets: Dict, edition: Dict):
             '''
         )
 
+def _assert_editions__ordered_by_issued(list_of_datasets, topic: str, sub_topic: str):
+    """this function will assert that the list of datasets
+     is ordered by issued, raises error if not"""
+
+    index  = 0
+
+    list_of_issued=[]
+    
+    for x in list_of_datasets[topic]:
```

Comment:
try and use descriptive variables please, "x" is only really acceptable inside a comprehension (its an _anonymous_ function). So something more like`for dataset_dict in list_of_datasets` etc, always think of what favours you can do for the next programmer to read your code.